js/acf/fields/markdown-editor.js

```diff
@@ -211,7 +211,7 @@
       var output = '[';
       output += jQuery('#simpleMDE-link-text').val();
       output += '](';
-      output += jQuery('#simpleMDE-link-url').val();
+      output += jQuery('#simpleMDE-link-url').val().replace('(', '%28').replace(')', '%29');
```

Comment:
```suggestion
      output += encodeURI(jQuery('#simpleMDE-link-url').val()).replace(/\(|\)/g, (char) => {
          return char === '(' ? '%28' : '%29';
      });
```

Reply:
Hvis vi først anvender encodeURI, burde vi gerne fange andre eventuelle fejl og så en enkelt replace, der søger på både `(` og `)` for forhåbentlig at gøre det mere overskueligt og "hurtigere" - tænker også det er lettere at tilføje ekstra chars i fremtiden, fremfor at appende uendeligt mange replace statements :-)